deploy-addresses/op-sepolia.json

```diff
@@ -1,4 +1,4 @@
 {
-  "UniversalRouter": "0x78e7F1502A9e4115dEbd6876E0AC4FaEBDB96880",
+  "UniversalRouterV1_2_NoV2Support": "0x78e7F1502A9e4115dEbd6876E0AC4FaEBDB96880",
```

Comment:
this is actually a router that supports v4 but deployed in october... so i think we should remove it probably

Reply:
oh really? it says Universal Router here https://sepolia-optimism.etherscan.io/address/0x78e7f1502a9e4115debd6876e0ac4faebdb96880#code

Reply:
looks like it's UniversalRouterV2?

Reply:
maybe we never deployed the older versions to OP Sepolia?